server/bin/gold/test-5.2.txt

```diff
@@ -28,36 +28,6 @@ len(actions) = 1
     }
```

Comment:
This is not the easiest set of changes to review.  In order to perform a proper review of this file one needs to `tar tvf server/bin/state/test-5.2.tar.xz` to see the starting state of the test.

There are two major changes here: removal of `DUPLICATE__NAME` tar balls and the removal of the `TODO` directory.

Really the only way to review this file is to just start with the tar ball state, and see what happens to each file in the hierarchy and what is reported in the output of the various tests.

Yes, painful.  Sorry.  :(